<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QDialog" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connection Eurêka</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMDP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Mot de Passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butConnection">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Connection 
 échouée !!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ID ou MDP invalide</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_erreur1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
    </font>
   </property>
   <property name="text">
    <string>Erreur:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
